specification/azurefleet/resource-manager/Microsoft.AzureFleet/stable/2024-11-01/azurefleet.json

```diff
@@ -0,0 +1,3581 @@
+          }
+        ]
+      }
+    },
+    "VMAttributes": {
+      "type": "object",
+      "description": "VMAttributes that will be used to filter VMSizes which will be used to build Fleet.",
+      "properties": {
+        "vCpuCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of vCpuCount specified from Min to Max. Must be specified if VMAttributes are specified, either Min or Max is required if specified."
+        },
+        "memoryInGiB": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of memory specified from Min to Max. Must be specified if VMAttributes are specified, either Min or Max is required if specified."
+        },
+        "localStorageSupport": {
+          "$ref": "#/definitions/VMAttributeSupport",
+          "description": "Specifies whether the VMSize supporting local storage should be used to build Fleet or not."
+        },
+        "localStorageInGiB": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of local storage in GB specified from Min to Max."
+        },
+        "localStorageDiskTypes": {
+          "type": "array",
+          "description": "The local storage disk types specified as a list. LocalStorageSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf localStorageSupport is \"Excluded\", this VMAttribute can not be used.",
+          "items": {
+            "$ref": "#/definitions/LocalStorageDiskType"
+          },
+          "x-ms-identifiers": []
+        },
+        "dataDiskCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of data disk count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "networkInterfaceCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "networkBandwidthInMbps": {
+          "$ref": "#/definitions/VMAttributeMinMaxDouble",
+          "description": "The range of network bandwidth in Mbps specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
+        },
+        "rdmaSupport": {
+          "$ref": "#/definitions/VMAttributeSupport",
+          "description": "Specifies whether the VMSize supporting RDMA (Remote Direct Memory Access) should be used to build Fleet or not."
+        },
+        "rdmaNetworkInterfaceCount": {
+          "$ref": "#/definitions/VMAttributeMinMaxInteger",
+          "description": "The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified."
```

Comment:
rdmaSupport should be set to \"Included\" or \"Required\" to use this VMAttribute. \nIf rdmaSupport is \"Excluded\", this VMAttribute can not be used." should be added to description for rdmaNetworkInterfaceCount.

Reply:
Updated.